### PR TITLE
Gough Book 5, 1-2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc [hunk 1]
```diff
@@ -62,8 +65,13 @@ Accordingly, about a month after this king`'s accession to the throne, George Wh
 Gilbert Latey and Alexander Parker waited upon him with the following address,
 and case of their friends:
 
+[.embedded-content-document.address]
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc:68]: Another funky letter, and I can't remember how we've done them!!! I'm sorry!!!

File: history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc [hunk 2]
```diff
@@ -130,8 +142,11 @@ from prisons, spoil and ruin.
...
 and his eternal happiness in that which is to come.
 
+[.signed-section-context-close]
 London, the 2nd of the 1st month called March, 1684-5.
 
+--
+
 Soon after the friends of London drew up a more copious state of their case and sufferings,
 addressed to the king and both houses of parliament,
 wherein they recapitulate their grievous sufferings
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc:152]: I can't leave a note on line 157, but a footnote starts there listing some things and there are paragraph breaks between each one, and I wasn't sure if I should leave them or delete them like that other footnote that had discourse in it.

File: history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc [hunk 3]
```diff
@@ -235,6 +250,10 @@ But being earnestly desirous to obtain the liberty of their imprisoned friends,
 the friends of London renewed their solicitations to the king for their liberation,
 and at length obtained the following warrant to Sir Robert Sawyer, attorney-general:
 
+[.embedded-content-document.legal]
```
Comment (maijapapaya) [inline history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc:253]: Another weird letter....ugh!

File: history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc (hunk 1 above)
Comment (jaredh159) [reply history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc:68]: yup, funky indeed.  I think i like how you did it here. 👍

File: history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc (hunk 3 above)
Comment (jaredh159) [reply history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc:253]: yeah, this is like that other one. i don't know what to do with it.  if there wasn't already a `sss` below, we could move it down. For lack of a better idea, let's go with what you did here. It looks about like the original did. 👍

**Pull request merged**